x-pack/plugin/rest-compatibility/src/main/java/org/elasticsearch/compat/CompatibleApiException.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.compat;
+
+public class CompatibleApiException extends RuntimeException {
```

Comment:
Do we need a new exception? Maybe we can just use `ElasticsearchStatusException`?

Reply:
we probably don't need it. will replace with `ElasticsearchStatusException`